### PR TITLE
D8CORE-3050 Added path redirect import module

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Added path redirect import module for csv uploads.

# Need Review By (Date)
- 2/25

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-3050 --no-update`
2. `composer update -n`
3. `drush cim -y`
4. test out the importer on `/admin/config/search/redirect/import`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -207,6 +208,9 @@
             "drupal/paragraphs": {
                 "https://www.drupal.org/project/paragraphs/issues/2901390": "https://www.drupal.org/files/issues/2020-06-25/paragraphs-2901390-51.patch"
             },
+            "drupal/path_redirect_import": {
```
Comment (imonroe) [inline composer.json:211]: should this be in the new `drupal-patches` module instead?

**Review by imonroe — CHANGES_REQUESTED** [review]
Looks like the patches here need to move; see also merge-conflict with composer.json

**Comment by imonroe** [comment]
I'm trying to install this per the instructions, but composer won't resolve it to installable packages.  I've tried this with a fresh install of the `2.x` branch of acsf-cardinalsites, and it's still having problems.

```
  Problem 1
    - Root composer.json requires drupal/domain_301_redirect dev-2954889-d9-ready as 1.0.0, it is satisfiable by drupal/domain_301_redirect[dev-2954889-d9-ready] from vcs repo (git https://git.drupalcode.org/issue/domain_301_redirect-2954889) but drupal/domain_301_redirect[dev-1.x, 1.0.0-alpha0, 1.x-dev (alias of dev-1.x)] from composer repo (https://packages.drupal.org/8) has higher repository priority. The packages with higher priority do not match your constraint and are therefore not installable. See https://getcomposer.org/repoprio for details and assistance.
  Problem 2
    - Root composer.json requires drupal/link_title_formatter dev-3174382-d9-ready as 1.1.0, it is satisfiable by drupal/link_title_formatter[dev-3174382-d9-ready] from vcs repo (git https://git.drupalcode.org/issue/link_title_formatter-3174382) but drupal/link_title_formatter[1.0.0, 1.1.0] from composer repo (https://packages.drupal.org/8) has higher repository priority. The packages with higher priority do not match your constraint and are therefore not installable. See https://getcomposer.org/repoprio for details and assistance.
  Problem 3
    - Root composer.json requires drupal/paranoia dev-3148370-d9-ready as 1.9.0, it is satisfiable by drupal/paranoia[dev-3148370-d9-ready] from vcs repo (git https://git.drupalcode.org/issue/paranoia-3148370) but drupal/paranoia[dev-1.x, 1.0.0-alpha1, 1.x-dev (alias of dev-1.x)] from composer repo (https://packages.drupal.org/8) has higher repository priority. The packages with higher priority do not match your constraint and are therefore not installable. See https://getcomposer.org/repoprio for details and assistance.
  Problem 4
    - Root composer.json requires drupal/response_code_condition dev-3141520-d9-ready as 1.0.0, it is satisfiable by drupal/response_code_condition[dev-3141520-d9-ready] from vcs repo (git https://git.drupalcode.org/issue/response_code_condition-3141520) but drupal/response_code_condition[dev-1.x, 1.0.0, 1.x-dev (alias of dev-1.x)] from composer repo (https://packages.drupal.org/8) has higher repository priority. The packages with higher priority do not match your constraint and are therefore not installable. See https://getcomposer.org/repoprio for details and assistance.
  Problem 5
    - Root composer.json requires su-sws/stanford_profile dev-D8CORE-3050 -> satisfiable by su-sws/stanford_profile[dev-D8CORE-3050].
    - su-sws/stanford_profile dev-D8CORE-3050 requires drupal/domain_301_redirect ^1.0-alpha0 -> found drupal/domain_301_redirect[dev-1.x, 1.x-dev (alias of dev-1.x)] but it conflicts with your root composer.json require (dev-2954889-d9-ready as 1.0.0).
```

**Comment by pookmish** [comment]
I updated the steps. it just requires a `--no-update` and then a full composer update to fix your issue.

**Review by imonroe — APPROVED** [review]
I got past the composer problems once I downgraded composer to 1.9.x.  Configs wouldn't import until I manually activated the module, but I'm not worried about that.  Got the configs imported and everything tests good.

**Pull request merged**